tests/system/hw_math/source/main.c

```diff
@@ -49,6 +235,8 @@ int main(int argc, char **argv)
     printf("qNaN: 0x%08lX\n", quiet_nan);
     printf("\n");
```

Comment:
Don't add (or remove) empty lines for no reason, they just add noise to the git history.